lib/elixir/lib/enum.ex

```diff
@@ -339,6 +339,9 @@ defmodule Enum do
   elements. If `leftover` is not passed at all or is `nil`, then the
   partial chunk is discarded from the result.
 
+  If `count` is greater than the number of elements in the enumerable
```

Comment:
yes, I added this as an afterthought. Indeed it's somewhat redundant with the previous paragraph but I thought it's worth to really make this corner case stand out (not only in examples). Previous paragraph mentions partial chunks which, perhaps technically correct, might be misleading. As long as the doctests showing this corner case are in, I think folks won't make a mistake I've recently made using this function.